src/test/java/com/exactpro/junit5/WithByteBufInvocationContext.java

```diff
@@ -13,49 +13,50 @@
 
     private final List<ByteBuf> arguments;
 
-    private StringBuilder displayName;
+    private String methodName;
 
     private WithByteBufMethodContext methodContext;
 
     WithByteBufInvocationContext(String methodName, WithByteBufMethodContext methodContext, List<ByteBuf> arguments) {
-        this.displayName = new StringBuilder(methodName);
+        this.methodName = methodName;
         this.arguments = arguments;
         this.methodContext = methodContext;
     }
 
     @Override
     public String getDisplayName(int invocationIndex) {
-        return formatName(arguments);
+        return formatName(methodName, arguments);
     }
 
     @Override
     public List<Extension> getAdditionalExtensions() {
         return singletonList(new WithByteBufParameterResolver(methodContext, this.arguments));
     }
 
-    private String formatName(Collection<ByteBuf> arguments) {
+    private String formatName(String methodName, Collection<ByteBuf> arguments) {
+        StringBuilder displayName = new StringBuilder(methodName);
         for (ByteBuf buf : arguments) {
-            appendFormattedBuffer(buf);
+            appendFormattedBuffer(displayName, buf);
         }
         return displayName.toString();
     }
 
-    private void appendFormattedBuffer(ByteBuf buf) {
+    private void appendFormattedBuffer(StringBuilder currentName, ByteBuf buf) {
         int readerIndex = buf.readerIndex();
         int readLimit = buf.writerIndex();
-        displayName.append(" [ ");
+        currentName.append(" [ ");
         while (readerIndex < readLimit) {
-            appendHexByte(buf.getByte(readerIndex++));
+            appendHexByte(currentName, buf.getByte(readerIndex++));
         }
-        displayName.append("]");
+        currentName.append("]");
     }
 
-    private void appendHexByte(int oneByte) {
-        if (oneByte == 0) {
-            displayName.append("00");
+    private void appendHexByte(StringBuilder currentName, int oneByte) {
```

Comment:
`aByte` is a better name

Reply:
there are uses of oneByte in other classes to, do you want to replace them all?

Reply:
It's better to do it separately